Migrate hosting docs to GitHub (#4166)
Scille/parsec-cloud: README.rst

Parsec is also available directly through PIP for both Linux and Windows with Python > 3.9 with the command:
``pip install parsec-cloud``
(or, if you need to specify Python 3 pip version, ``pip3 install parsec-cloud``)

Self-hosted
-----------

> If you want to self-host a parsec server, take a look a the documentation `Hosting the server <https://github.com/Scille/parsec-cloud/blob/master/docs/HOSTING.md>`_

Reviewer: ```suggestion
If you want to self-host a Parsec server, take a look at `Hosting a Parsec server <https://github.com/Scille/parsec-cloud/blob/master/docs/HOSTING.md>`_
```
